dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Account/OfficialName/Evidence.cshtml

```diff
@@ -25,9 +26,9 @@
                 <li>deed poll</li>
             </ul>
 
-            <p>Your scan must be readable and smaller than @Evidence.MaxFileSizeMb MB.</p>
+            <p>Your scan must be readable and smaller than @BaseEvidencePage.MaxFileSizeMb MB.</p>
 
-            <govuk-file-upload asp-for="EvidenceFile" input-accept=".csv, .jpg, .jpeg">
+            <govuk-file-upload asp-for="EvidenceFile" input-accept=".pdf, .jpg, .jpeg">
```

Comment:
This was csv in the OfficialName change ticket, but I don't think that's correct so have updated here and in the ticket